docs/setup-ci.md

```diff
@@ -0,0 +1,102 @@
+---
+id: setup-ci
+title: Setup Continuous Integration Pipeline for an RNW App
+---
+
+This guide will help you get started on setting up your very first continuous integration pipeline for a React Native for Windows app.
+
+## Setting Up a Continuous Integration Pipeline using Github Actions
+
+When done developing your app, it's good practice to setup a CI pipeline with automated build and tests to avoid any future regressions. There are many services available for setting up a CI pipeline. We'll use [GitHub Actions](https://docs.github.com/actions/getting-started-with-github-actions/about-github-actions) as an example here since it doesn't require any extra account setup if you are already hosting your code on GitHub, also the default VM image has all the tools we needed pre-installed.
+
+The VM images supported by GitHub Actions CI/CD can be found [here](https://github.com/actions/virtual-environments#github-actions-virtual-environments), check the pre-installed tools and compare them with [React Native Windows development dependencies](https://microsoft.github.io/react-native-windows/docs/rnw-dependencies), find the image that meets the requirements.
+
+Next you need to create a YAML file for GitHub Actions, the basic steps are:
+- Checkout code and setup the environment
+```yaml
+    - uses: actions/checkout@v2
+      name: Checkout Code
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '14'
+
+    - name: Setup MSBuild
+      uses: microsoft/setup-msbuild@v1.0.2
+      with:
+        vs-version: 16.8
+
+    - name: Install node modules
+      run: yarn --frozen-lockfile
+
+    - name: yarn build
+      run:  yarn build
+```
+- Build and run the project
+```yaml
+    - name: Run Windows x64 release
+      run: npx react-native run-windows --arch x64 --release --no-packager --logging --deploy-from-layout
```

Comment:
1) isn't no-packager the default for release anyway? (if not, I think it should be)
2) see my comment on the other PR re: deploy from layout, if now it won't work otherwise from the CLI, then it should be the default for release too, and maybe we add a new "create package" option to the CLI